### ISSUE TITLE
Exit early from exec

### ISSUE THREAD

**Issue by corbinu** [sharkdp/fd#224]
First of all love fd is an amazing project!

I was using it recently to exec a sed command and made a mistake. Noticed that `Ctrl+C` to exit early wasn't working

**Comment by sharkdp** [comment]
Thank you very much for the feedback!

Indeed, that looks like a problem.

One way to reproduce this:
``` bash
> mkdir test; cd test
> touch {1..100}
> fd -x sleep
# try to break out via Ctrl-C
```

**Comment by Doxterpepper** [comment]
Sorry @sharkdp, looks like this is the fault of the ctrl-c handler agian 🙃 . The ctrl-c handler I added months ago didn't account for exiting from exec. For some reason I thought colored output would not be enabled when the exec flag was given. I'll have a PR for this in a few.

**Comment by sharkdp** [comment]
> Sorry @sharkdp, looks like this is the fault of the ctrl-c handler agian  .

Nothing to excuse for - thank you very much for finding the bug and fixing it so fast!

**Comment by sharkdp** [comment]
Closed via #225

**Comment by sharkdp** [comment]
Fix released in v6.3.0

**Comment by corbinu** [comment]
Thanks folks :)
